torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4376,21 +4376,21 @@ def test_post_localSGD_optimizer_parity(self, grad_is_view=False):
             post_localSGD_opt = post_localSGD_optimizer.PostLocalSGDOptimizer(
                 params=post_localSGD_net.parameters(),
                 optimizer_class=torch.optim.SGD,
-                averager=averagers.PeriodicModelAverager(post_localSGD_net.parameters(), period=period, warmup_steps=warmup_steps),
+                averager=averagers.PeriodicModelAverager(period=period, warmup_steps=warmup_steps),
                 lr=learning_rate,
             )
 
             input = torch.randn(dist.get_world_size() * 2, 2).cuda()
             target = torch.randn(dist.get_world_size() * 2, 4).cuda()
             loss_fn = nn.MSELoss()
 
-            for idx in range(20):
+            for step in range(20):
```

Comment:
nit: `for _ in range(20)` since it is not used?

Reply:
Sure. Previously used step and then forgot to remove it here. Hopefully this can be captured by linter in the future.